routes/web3relay.js

```diff
@@ -81,11 +81,37 @@ exports.data = function(req, res){
       } else {
         var ttx = tx;
         ttx.value = etherUnits.toEther( new BigNumber(tx.value), "wei");
+        ttx.gasPriceGwei = new BigNumber(tx.gasPrice).div(etherUnits.getValueOfUnit('gwei')).toString(10);
+        ttx.gasPrice = etherUnits.toEther( new BigNumber(tx.gasPrice), "wei");
+        //get TxReceipt status & gasUsed
+        var receipt = web3.eth.getTransactionReceipt(txHash);
+        if (receipt.status != "0x0"){
+          ttx.status = "Success";
+        }
+        else{
+          ttx.status = "Failure";
+        }
+        ttx.gasUsed = receipt.gasUsed;
         //get timestamp from block
         var block = web3.eth.getBlock(tx.blockNumber, function(err, block) {
           if (!err && block)
             ttx.timestamp = block.timestamp;
-          ttx.isTrace = (ttx.input != "0x");
+          ttx.isTrace = false;
+          if (ttx.input != "0x") {
+            // check if input data is a string or contact data
+            if (ttx.to != null) {
+              var bytecode = web3.eth.getCode(ttx.to);
+              if (bytecode == "0x") {
+                // decode string data
```

Comment:
Can you restyle this section like etherscan?

![default](https://user-images.githubusercontent.com/44437404/51768840-cb9b4280-2124-11e9-9e88-1804b890eb86.PNG)